src/Pos/Delegation/Class.hs

```diff
@@ -56,9 +54,6 @@ data DelegationWrap = DelegationWrap
       -- a good approximation.
     , _dwEpochId           :: EpochIndex
       -- ^ Epoch index 'DelegationWrap' is correct in relation to.
-    , _dwThisEpochPosted   :: HashSet PublicKey
```

Comment:
Because there's no need for it. Previously we were checking if PSK is legal using this set before putting it to mempool. Now we just do a single read from DB. I think it can be even faster.